product/src/test/e2e/createProduct.test.js

```diff
@@ -0,0 +1,41 @@
+import { productExample } from "../data/products.js";
+import { app } from "../../app.js";
+import request from "supertest";
+import client from "../../repositories/databaseClients.js";
+
+describe("Criação de produto", () => {
+  afterAll(async() => {
+    await client.close();
```

Comment:
why are you closing the connection here?

Reply:
here it was my lack of attention, the statement asked to clean the database and not close it, I already removed that line of code.